inc/class-sitemaps-index.php

```diff
@@ -10,27 +10,27 @@
  */
 
 /**
- * Class Core_Sitemaps_Index.
+ * Class Sitemaps_Index.
  * Builds the sitemap index page that lists the links to all of the sitemaps.
  *
  * @since 5.5.0
  */
-class Core_Sitemaps_Index {
+class Sitemaps_Index {
```

Comment:
Same with classes, those should be `WP_Sitemaps_*`.

Reply:
👍